resources/languages/it/rules/time.clj

```diff
@@ -17,31 +17,31 @@
   ;;
 
   "named-day"
-  #"(?i)luned[ìi]|lun?\.?"
+  #"(?i)luned[iì]|lun\.?"
   (day-of-week 1)
 
   "named-day"
-  #"(?i)marted[iì]|mar?\.?"
```

Comment:
Yep that makes more sense to me - "ma" is so common that this seemed aggressive.